modules/Microsoft.DevTestLab/labs/costs/deploy.bicep

```diff
@@ -0,0 +1,189 @@
+@sys.description('Conditional. The name of the parent lab. Required if the template is used in a standalone deployment.')
+param labName string
+
+@allowed([
+  'Custom'
+  'CalendarMonth'
+])
+@sys.description('Required. Reporting cycle type.')
+param cycleType string
+
+@sys.description('Optional. Tags of the resource.')
+param tags object = {}
+
+@sys.description('Conditional. Reporting cycle start date in the zulu time format (e.g. 2023-12-01T00:00:00.000Z). Required if cycleType is set to "Custom".')
+param cycleStartDateTime string = ''
+
+@sys.description('Conditional. Reporting cycle end date in the zulu time format (e.g. 2023-12-01T00:00:00.000Z). Required if cycleType is set to "Custom".')
+param cycleEndDateTime string = ''
+
+@allowed([
+  'Enabled'
+  'Disabled'
+])
+@sys.description('Optional. Target cost status. Default is "Enabled".')
```

Comment:
```suggestion
@sys.description('Optional. Target cost status.')
```

Reply:
Updated and reflected it across all parameters